Makefile.maker.yaml

```diff
@@ -12,6 +12,8 @@ golangciLint:
 githubWorkflow:
   global:
     ignorePaths: ["README.md"]
+  autoUpdate:
```

Comment:
"autoUpdate" sounds like it would enable auto-updating of dependencies, not auto-regenerating the Makefile. I'd prefer something like "autoRerender".